src/pages/proposal/detail/index.tsx

```
@@ -258,40 +257,74 @@ const Detail = () => {
+                  color="text.secondary"
+                  sx={{
+                    display: '-webkit-box',
+                    WebkitLineClamp: 2,
+                    textOverflow: 'ellipsis',
+                    overflow: 'hidden',
+                    WebkitBoxOrient: 'vertical',
+                    textAlign: 'start',
+                  }}
                   variant="caption"
-                  color="grey.400"
                   dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)